store.go

```diff
@@ -240,3 +265,79 @@ func (s *SafeConfigStore) Size() int {
 	defer s.mu.Unlock()
 	return s.c.Size()
 }
+
+// LogableConfigStore implements the ConfigStore interface and provides a store
+// that is safe to read and write from multiple goroutines.
+type LogableConfigStore struct {
+	c  *DefaultConfigStore
```

Comment:
This should probably be a `ConfigStore`. That way a `LogableConfigStore` can be used to wrap any `ConfigStore` that you might want to debug